source/connect/connections.txt

```diff
@@ -0,0 +1,85 @@
+.. _connections-tab:
+
+===================
+Connections Sidebar
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The :guilabel:`Connections Sidebar` shows MongoDB deployments that
+you can interact with through MongoDB Compass. 
+
+.. note::
+
+   .. include:: /includes/multiple-connections.rst
+
+Connections Sidebar
+-------------------
+
+The :guilabel:`Connections` sidebar on the left of the Compass window
+contains the following options:
+
+- :guilabel:`Active Connections` menu
+- :guilabel:`Connections` menu
+- :icon-lg:`Plus` icon to add new connections
+
+Active Connections Menu
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To interact with a MongoDB deployment, you must activate the connection.
+When you activate a connection, the connection name appears in the
+:guilabel:`Active Connections` menu at the top of
+:guilabel:`Connections` sidebar.
+
+If you click the :icon-fa5:`ellipsis-h` button on the right of a
+connection name, a menu with the following items opens:
+
+- View performance metrics
+- Show connection info
+- Copy connection string
+- Favorite 
+- Disconnect
+
+If the connection is already a favorite connection, the
+option to Unfavorite appears in the menu. 
```

Comment:
[question] Just to double check, are we leaving "Unfavorite" capitalized to indicate what the button will say if the connection's already favorited? If so, we'll want to use the guilabel tag here!